### PR TITLE
run tests in docker container

### PR DESCRIPTION
**Pull request by msokoloff1**

### CODE REVIEW

File: test/seed/seed_bbox.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+import pathlib
+import os
 
 from labelbox import Client, DataRow, OntologyBuilder, Tool, LabelImport
 from labelbox.data.serialization import NDJsonConverter, LBV1Converter
 
 
 def load_labels():
-    with open("../assets/proj_ckq778m4g0edr0yao004l41l7_export.json") as f:
+    with open(os.path.join(pathlib.Path(__file__).parent.resolve(), "../assets/proj_ckq778m4g0edr0yao004l41l7_export.json")) as f:
```
Comment (gszpak) [inline test/seed/seed_bbox.py:11]: nit: `pathlib.Path(__file__).parent / Path("assets/proj_ckq778m4g0edr0yao004l41l7_export.json")`

File: test/seed/seed_ner.py [hunk 2]
```diff
@@ -125,3 +125,49 @@ def main(client):
 if __name__ == '__main__':
     client = Client()
     main(client)
+
+
+
+"""
+# On a fresh ubuntu instance:
+
```
Comment (gszpak) [inline test/seed/seed_ner.py:133]: No pls let's not do it. We should list requirements in the docs, but people have to figure out how to install them themselves.

File: README.md [hunk 3]
```diff
@@ -138,3 +138,11 @@ Key terms:
 3. Add the job to a pipeline in `coordinator/config.py`
     * Update `pipelines` to include the new workflow
     * Add the new pipeline name `PipelineName` type as a Literal
+
+
+### Testing
+
+* To run any of the seed scripts you can use the `model_training_tests` container
+    * Run: `docker-compose run model_training_tests python3 tests/seed/<script>.py`
```
Comment (gszpak) [inline README.md:146]: nit: could we include the default command in the dockerfile

File: test/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,6 @@
+FROM python:3.9-slim-buster
+RUN apt-get update -y && apt-get install -y libgl1-mesa-glx libglib2.0-0
+WORKDIR test
+COPY requirements.txt .
```
Comment (gszpak) [inline test/Dockerfile:4]: Shouldn't we also copy `test/seed` dir?

File: docker-compose.yaml [hunk 5]
```diff
@@ -90,3 +90,18 @@ services:
...
+    volumes:
+      - type: bind
+        source: ${GOOGLE_APPLICATION_CREDENTIALS}
+        target: ${GOOGLE_APPLICATION_CREDENTIALS}
+      - type: bind
+        source: ./
+        target: /test/
+    environment:
+      LABELBOX_API_KEY: ${LABELBOX_API_KEY}
+      GOOGLE_APPLICATION_CREDENTIALS: ${GOOGLE_APPLICATION_CREDENTIALS}
```
Comment (gszpak) [inline docker-compose.yaml:107]: If it's a one time thing, I would rather write a simple bash script that:
- takes seed script as a param
- builds docker image
- runs docker image

File: test/Dockerfile (hunk 4 above)
Comment (msokoloff1) [reply test/Dockerfile:4]: we are mounting the dir so changes don't require rebuilding the container

File: docker-compose.yaml (hunk 5 above)
Comment (msokoloff1) [reply docker-compose.yaml:107]: I was going to use this to run tests too. But yeah, i'll just build and run a seed container via a test

**Review by gszpak — APPROVED** [review]

**Pull request merged**